app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -6,6 +6,10 @@ import 'dart:async';
 
 import 'package:appengine/appengine.dart';
 import 'package:cocoon_service/src/request_handling/exceptions.dart';
+import 'package:cocoon_service/src/service/github/labeled_pull_requests_with_reviews.data.gql.dart';
+import 'package:cocoon_service/src/service/github/labeled_pull_requests_with_reviews.op.gql.dart';
+import 'package:cocoon_service/src/service/github/labeled_pull_requests_with_reviews.var.gql.dart';
+import 'package:cocoon_service/src/service/github/schema.public.schema.gql.dart' show StatusState, CommentAuthorAssociation, PullRequestReviewState;
```

Comment:
How about these import, are they auto added to the file? If not, do we need to run the build first, and then import then here?